package.json

```diff
@@ -19,11 +19,12 @@
     "url": "https://github.com/cloudlibz/nodecloud-cli/issues"
   },
   "bin": {
-    "nc": "./bin/nc.js"
+    "nodec": "./bin/nc.js"
```

Comment:
Any particular reason to change nc to nodec ?. @MarkChenYutian

Reply:
sorry I forgot to delete that. There is a CLI tool called nc on Mac so I changed that on my laptop to avoid conflict.